src/toast/ops/flag_intervals.py

```diff
@@ -20,7 +20,8 @@ class FlagIntervals(Operator):
     with shared flags.  The view_mask trait is a list of tuples.  Each tuple contains
     the name of the view (i.e. interval) to apply and the bitmask to use for that
     view.  For each interval view, flag values in the shared_flags object are bitwise-
-    OR'd with the specified mask for samples in the view.
+    OR'd with the specified mask for samples in the view.  If the name of the view is
+    prefixed with '~' or '-', the bitmask is applied to all samples outside the view.
```

Comment:
You mention using `-` here in addition to `~`, but the observation view manager only considers `~`.  Did I miss something?

Reply:
That was the original idea but really only `~` makes sense.